tests/io/vasp/test_sets.py

```diff
@@ -239,13 +241,13 @@ def test_nelect(self):
         coords = [[0] * 3, [0.5] * 3, [0.75] * 3]
         lattice = Lattice.cubic(4)
         struct = Structure(lattice, ["Si", "Si", "Fe"], coords)
-        assert self.set(struct).nelect == 16
-        assert MPRelaxSet(struct).nelect == 22
+        assert self.set(struct).nelect == 18.758
+        assert MPRelaxSet(struct).nelect == 27.166
```

Comment:
Why did these values change? Are the fake POTCARs different even in the valence electrons? Could that have downstream implications? Like if we ever tested the same way we do in `atomate2`, we no longer can use actual VASP output as references files?

Reply:
The POTCARs have all non-string fields randomized, including the valence. Wouldn't be too challenging to change this. Another option is to simply change the data in the body of the POTCAR, since the header data is included in vasprun.xml and OUTCAR. And then I'm guessing not copyright protected, but would need confirmation from vasp

Reply:
Hi @janosh really sorry this took me forever to get back to! I've modified the POTCAR scrambling to only change data that doesn't appear in OUTCAR. That's basically everything after the line "Error from kinetic energy argument (eV)"

Pending tests passing, should be good to go!

Reply:
And to just be clear, that means properties like the number of electrons, nuclear charge, core radii, etc. are unaffected. Only the values of the pseudopotential and the kinetic energy corrections (which I don't think are meaningful to most)